conf.yaml

```diff
@@ -1106,6 +1107,22 @@ contents:
                         repo:   apm-agent-go
                         path:   CHANGELOG.asciidoc
                         exclude_branches:   [ 0.5 ]
+                  - title: APM iOS Agent
+                    prefix: swift
+                    current: main
```

Comment:
@bryce-b I changed this to `main` because there is no `0.x` branch in the apm-ios-agent repo yet. The `current` variable, `branches` array, and `live` array, must match branch names in the associated repo

Reply:
I also moved this entire section up to maintain alphabetical ordering for the APM agents